cli/popper/resources/.search_sources.yml

```diff
@@ -0,0 +1,6 @@
+organizations:
+- popperized
+- actions
+repositories:
```

Comment:
we can add the repos that are currently listed [here](https://github.com/boyney123/github-actions/tree/master/src/actions).